src/SDKs/ManagementGroups/Management.ManagementGroups/Management.ManagementGroups.csproj

```diff
@@ -5,11 +5,11 @@
   
   <PropertyGroup>
 		<Description>Microsoft Azure Management Groups Library</Description>
-		<Version>1.1.0-preview</Version>
+		<Version>1.2.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.ManagementGroups</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.ManagementGroups</PackageId>
 		<PackageTags>Microsoft Azure Management Groups;Management Groups;Groups;</PackageTags>
-		<PackageReleaseNotes>This is a public preview release of the Azure Management Groups SDK</PackageReleaseNotes>
+		<PackageReleaseNotes>This is a general availibility release of the Azure Management Groups SDK</PackageReleaseNotes>
```

Comment:
So, is this a stable release?

Reply:
Yes. We are GAing the end of this month.

Reply:
In that case, you will need to remove the `-preview` from the package version.

Reply:
Actually, if removing the preview tag is not a requirement for GAing a service we would like to keep it. Please let me know if we can keep the preview tag.